tests/bgp/test_bgp_fact.py

```diff
@@ -5,14 +5,11 @@
     pytest.mark.device_type('vs')
 ]
 
-def test_bgp_facts(duthosts, enum_dut_hostname, enum_asic_index):
-    """compare the bgp facts between observed states and target state"""
 
-    duthost = duthosts[enum_dut_hostname]
+def test_bgp_facts(duthosts, enum_frontend_dut_hostname, enum_asic_index):
+    """compare the bgp facts between observed states and target state"""
 
-    # Check if duthost is 'supervisor' card, and skip the test if dealing with supervisor card.
```

Comment:
what verifications happen on a supervisor for test_bgp_facts ?

Reply:
There are no BGP session on supervisor card